src/containers/App.js

```diff
@@ -1,7 +1,8 @@
-import React from 'react'
+import React, {Component} from 'react'
 import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom'
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
-import Sidebar from '../components/Sidebar'
+import { Snackbar } from 'material-ui';
+import { Sidebar } from '../components'
 import AssetList from './AssetList'
 import AssetForm from './AssetForm'
 import Static from '../components/Static'
```

Comment:
Rename Static to StaticRoute? From this file, it's not clear that it now acts as a a Route.